source/usr/local/emhttp/plugins/lxc/LXCSnapshots.page

```diff
@@ -0,0 +1,50 @@
+Menu="LXC:1"
+Title="Snapshots"
+---
+<?php
+$avail_containers = explode("\n", shell_exec("lxc-ls"));
+
+foreach ($avail_containers as $container) {
+    if (!empty($container)) {
+?>
+        <h2><?= $container ?></h2>
+        <br>
+        <table id="snapshot_table" class="tablesorter two shift">
+            <thead>
+            <tr>
+                <th>Name</th>
+                <th>Date/Time</th>
+                <th>Actions</th>
+            </tr>
+            </thead>
+            <tbody align="left">
+<?
+        $snapshots = trim(shell_exec("/usr/local/emhttp/plugins/lxc/include/exec.sh get_snapshot " . $container));
+        if (strpos($snapshots, "snapshots") === false) {
+            $new = explode("\n", $snapshots);
+            foreach ($new as $new_cont) {
+                $sorted = explode(" ", $new_cont);
+                $delete_snapshot_button="<form id=\"snapshot\" method=\"post\">
+<input hidden type = \"text\" name=\"CONTname\" value=\"$container\">
+<input hidden type = \"text\" name=\"SNAPname\" value=\"$sorted[0]\">
+<input type=\"submit\" name=\"snapshotDELETE\" value=\"Delete Snapshot\">
+</form>";
+                $formatted_date = str_replace(":", "_", $sorted[2]);
+                echo "<tr style='height: 50px;'><td>$sorted[0]</td><td>$formatted_date $sorted[3] </td><td style='display: flex;'><a href='LXCRestoreSnapshot?cont=" . $container . "&snap=" . $sorted[0] . "&date=" . $formatted_date . "_" . $sorted[3] . "' class='button'>Create Container From Snapshot</a>" . $delete_snapshot_button . "</td></tr>";
```

Comment:
Don't know if that change is necessary, what do you thing about that? @JmzTaylor

Reply:
Technically I think its semantics but better clarification doesn't hurt anyone.